src/receiving_queue/ReceivingQueue.jsx

```diff
@@ -115,9 +166,12 @@ const ReceivingQueue = () => {
         })
         .catch((e) => {
           enqueueSnackbar(e.message, snackbarVariants.error);
+        })
+        .finally(() => {
+          reloadQueueData();
```

Comment:
Do we need to reload if we caught an error?

Reply:
I was thinking yes, in case other clients make changes, we can retrieve the latest in all cases.